CODEOWNERS

```diff
@@ -12,6 +12,8 @@
 #
 /packages/spec-haskell/ @hellwolf
 /packages/ethereum-contracts/ @hellwolf @0xdavinchee
+/packages/ethereum-contracts/contracts @hellwolf @0xdavinchee
```

Comment:
piggybacking CODEOWNERS updates here.

It occurs to me @d10r should gatekeeping the changes to scripts instead.

Reply:
I'd prefer gatekeeping the contracts tbh (contracts dir only, excluding tests), I feel like we're being too liberal with changes there.
Regarding scripts, there's those for ops (gov & other) which I feel qualified to gatekeep, and those mainly used in tests (inside and outside monorepo). I don't think I'm best qualified to gatekeep the latter.
Probably we should try to better categorize the scripts (into different dirs?) along that line.
Maybe related: I've in the past wondered what should go into `tasks` and what should go into `scripts`.

Reply:
Everyone should still review the code diligently, it's up to the codeowner to facilitate that.